flask/sessions.py

```diff
@@ -330,6 +342,16 @@ def open_session(self, app, request):
         except BadSignature:
             return self.session_class()
 
+    def regenerate_session(self, app, session):
+        # There isn't anything related to a session id so we do nothing
+        pass
+
+    def destroy_session(self, app, session):
+        # Empty store contents
+        session.clear()
```

Comment:
No, we shouldn't implement anything in the base class. When destroying a session we need to both destroy the cookie and its underlying store value. Otherwise, a user could "log out" (destroy their browser cookie) but an attacker could reuse the browser cookie and still access their session (since the underlying store data exists).